### PR TITLE
I3 35 merchant orders

### LINKED ISSUE #38: User Story 35, Merchant Dashboard displays Orders
As a merchant employee
When I visit my merchant dashboard ("/merchant")
If any users have pending orders containing items I sell
Then I see a list of these orders.
Each order listed includes the following information:
- the ID of the order, which is a link to the order show page ("/merchant/orders/15")
- the date the order was made
- the total quantity of my items in the order
- the total value of my items for that order

### PR DESCRIPTION
**Pull request by D-Lessenden**
I had to create a column "merchant_id" for users. This tied a merchant user to the merchant. 

Merchant dashboard now has a list of items if they have an item in an order with the status of pending.

As a consequence I had to update the following spec tests to add the column in the before each block
spec/features/navigation_spec.rb
spec/features/users/login_spec.rb

### CODE REVIEW

File: app/controllers/merchant/dashboard_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Merchant::DashboardController < Merchant::BaseController
 
   def show
+    @orders = Order.where(status: :pending).joins(:items).where(items: {merchant_id: current_user.merchant.id}).distinct
   end
```
Comment (PhilipDeFraties) [inline app/controllers/merchant/dashboard_controller.rb:5]: I think this should be abstracted to the model

**Review by PhilipDeFraties — CHANGES_REQUESTED** [review]
I'm pretty sure Tim or Bryan would tell us to abstract the code from the show action to the model

File: app/controllers/merchant/dashboard_controller.rb (hunk 1 above)
Comment (dhalverson) [reply app/controllers/merchant/dashboard_controller.rb:5]: I agree, this prob should be handled and tested in the model/model tests

Comment (D-Lessenden) [reply app/controllers/merchant/dashboard_controller.rb:5]: My mentor helped me build this and thought that the logic belonged in the controller. 

Why do you think it should be in the model?

**Review by PhilipDeFraties — APPROVED** [review]

**Pull request merged**